solutions/usaco-760.mdx

```diff
@@ -117,4 +120,34 @@ int main(){
 ```
 
 </CPPSection>
+	
+<PySection>
```

Comment:
since the two solutions are different, put them in separate languagesections (and move time complexity outside languagesections)